packages/api/src/sequelize/migrations/2025-01-19_00_create-hie_directory_view-with-fts.ts

```diff
@@ -0,0 +1,67 @@
+import { QueryTypes } from "sequelize";
+import type { Migration } from "..";
+
+const tableName = "cq_directory_entry_new";
+const hieViewName = "hie_directory_view";
+const cqViewName = "cq_directory_entry_view";
+const columnName = "search_criteria";
+
+const createSearchCriteriaColumnSql = `
+alter table ${tableName}
+add ${columnName} tsvector
+generated always as	(
+	to_tsvector('english', coalesce(id, '')) || ' ' ||
+	to_tsvector('english', coalesce(name, '')) || ' ' ||
+	to_tsvector('english', coalesce(root_organization, '')) || ' ' ||
+	to_tsvector('english', coalesce(address_line, '')) || ' ' ||
+	to_tsvector('english', coalesce(city, '')) || ' ' ||
+	to_tsvector('english', coalesce(state, '')) || ' ' ||
+	to_tsvector('english', coalesce(zip, ''))
+) stored;
+`;
+
+const createGinIndexSql = `
+CREATE INDEX IF NOT EXISTS ${tableName}_${columnName}_idx 
+ON ${tableName} 
+USING gin(${columnName});
+`;
+
+const dropCqViewSql = `DROP VIEW IF EXISTS ${cqViewName};`;
+const createCqViewSql = `CREATE VIEW ${cqViewName} AS SELECT * from ${tableName};`;
```

Comment:
Why are we recreating the CQ dir view? Note it gets recreated daily from the CQ dir rebuild logic.

Reply:
I found that on my local machine, the views built on top of the cq _new table with the new `search_criteria` column to were not picking up the new column. The views had been defined with select * statements, it seemed the actual definition was defined column by column, and I needed to drop and recreate the view